ospd/ospd.py

```diff
@@ -941,15 +941,20 @@ def get_severities_vt_as_xml_str(  # pylint: disable=unused-argument
         """
         return ''
 
-    def get_vt_xml(self, vt_id: str):
+    def get_vt_iterator(self) -> Iterable:
+        """ Return iterator object for getting elements
+        from the VTs dictionary. """
+        return iter(self.vts.items())
+
```

Comment:
I am really not sure about this method in several kinds.

1. Maybe we should already allow to filter the list of returned vts
2. No need to use the items for the iterator. The Vts implements the __iter__ protocol. Therefore it should return just `iter(self.vts)`
3. To avoid adding more and more methods to the daemon class we should access the "data" classes directly.

Personally I would be in favor of adding a more flexible method to the Vts class
```python
def get_filtered_vts(self, filter_func: Callable[Dict[str, str]]) -> Iterator[Dict[str,str]]:
  filter(filter_func, self)
```
which can be used in a command as

```
self.daemon.vts.get_filtered_vts(lambda vt: vt.vtd_id in vts_selection)
```
(see https://docs.python.org/3/library/functions.html#filter for details about filter)

Reply:
Regarding get filtered vts, I will do it with another PR.